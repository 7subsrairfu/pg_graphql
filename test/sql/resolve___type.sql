select jsonb_pretty(
    gql.dispatch($$
    {
      __type(name: "Account") {
        kind
        fields {
            name
        }
      }
    }
    $$)
);
