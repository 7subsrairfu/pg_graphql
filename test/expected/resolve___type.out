select jsonb_pretty(
    gql.dispatch($$
    {
      __type(name: "Account") {
        kind
        fields {
            name
        }
      }
    }
    $$)
);
                  jsonb_pretty                   
-------------------------------------------------
 {                                              +
     "data": {                                  +
         "__type": {                            +
             "kind": "OBJECT",                  +
             "fields": [                        +
                 {                              +
                     "name": "nodeId"           +
                 },                             +
                 {                              +
                     "name": "email"            +
                 },                             +
                 {                              +
                     "name": "encryptedPassword"+
                 },                             +
                 {                              +
                     "name": "createdAt"        +
                 },                             +
                 {                              +
                     "name": "updatedAt"        +
                 },                             +
                 {                              +
                     "name": "id"               +
                 },                             +
                 {                              +
                     "name": "blogs"            +
                 }                              +
             ]                                  +
         }                                      +
     },                                         +
     "errors": [                                +
     ]                                          +
 }
(1 row)

